src/include/daos_pool.h

```diff
@@ -94,24 +100,25 @@ struct daos_rebuild_status {
 	/** errno for rebuild failure */
 	int32_t			rs_errno;
 	/**
-	 * rebuild is done or not, it is valid only if @rs_version is non-zero
+	 * rebuild state, DRS_COMPLETED is valid only if @rs_version is non-zero
 	 */
-	int32_t			rs_done;
+	int32_t			rs_state;
```

Comment:
I think we need to keep rs_done and put rs_state where rs_padding32 is otherwise, we are breaking binary compatibility.   If we do it the way we do here, this is a major version change.   If we do it using the padding32 field and maintain the old rs_done as it was before, we can just update the minor API version.

These are found in the SConstruct
API_VERSION_MINOR = "0"

Reply:
Not sure how this could not break binary compatibility by magic number, is this struct only used by dmg utility you mean?